src/NetscriptJSEvaluator.ts

```diff
@@ -160,7 +158,12 @@ function generateLoadedModule(script: Script, scripts: Map<ScriptFilePath, Scrip
     // servers; it will be listed under the first server it was compiled for.
     // We don't include this in the cache key, so that other instances of the
     // script dedupe properly.
-    const adjustedCode = newCode + `\n//# sourceURL=${script.server}/${script.filename}`;
+    const sourceURL = `bitburner:///${script.server}/${script.filename}`;
```

Comment:
What's up with this fake `bitburner:` URI scheme?

Reply:
I modelled it after the `webpack://`/`webpack-internal://` scheme. I also chose not to put the hostname of the script in host part of the URL I give here. Doing what I did here leads to `bitburner://` showing up as a new entry in the devtools, with directories for each server.

Reply:
I *think* the URL is only really relevant for in the developer tools, and perhaps in tracebacks, so we can change this pretty freely. I experimented with a number of things, and landed on this as a first pass.